apps/web/public/static/locales/en/common.json

```diff
@@ -605,7 +605,7 @@
   "hide_book_a_team_member": "Hide Book a Team Member Button",
   "hide_book_a_team_member_description": "Hide Book a Team Member Button from your public pages.",
   "danger_zone": "Danger zone",
-  "account_deletion_cannot_be_undone":"Careful. Account deletion cannot be undone.",
+  "account_deletion_cannot_be_undone":"Be Careful. account deletion cannot be undone.",
```

Comment:
Shouldn't "account" be "Account" to match other locales like `disband_org_confirmation_message` ?

Reply:
@supalarry  Done